doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,364 @@
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
+
+         Depending on your connection, this might take some time.
+      #. Run the following command in your terminal:
+
+         .. parsed-literal::
+            :class: highlight
+
+            $HOME/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "$HOME/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This makes west and other development tools in the |NCS| toolchain environment available in the same shell session.
+
+         .. caution::
+            When working with west in the customer sampling release version of |NCS|, you must always use this shell window.
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+   .. tab:: macOS
+
+      Follow these steps:
+
+
+      1. Open a terminal window.
+      #. Install `Homebrew`_:
+
+         .. code-block:: bash
+
+            /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+      #. Use ``brew`` to install the required dependencies:
+
+         .. code-block:: bash
+
+            brew install cmake ninja gperf python3 ccache qemu dtc wget libmagic
+
+         Ensure that these dependencies are installed with their versions as specified in the :ref:`Required tools table <req_tools_table>`.
+         To check the installed versions, run the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+             brew list --versions
+
+      #. Download and run the :file:`bootstrap-toolchain.sh` installation script file using the following command:
+
+         .. parsed-literal::
+            :class: highlight
+
+            curl --proto '=https' --tlsv1.2 -sSf https://files.nordicsemi.com/artifactory/swtools/external/scripts/bootstrap-toolchain.sh | NCS_TOOLCHAIN_VERSION=v2.6.99-cs2 sh
+
+         Depending on your connection, this might take some time.
+
+         .. note::
+            On macOS, the install directory is :file:`/opt/nordic/ncs`.
+            This means that creating the directory requires root access.
+            You will be prompted to grant the script admin rights for the creation of the folder on the first install.
+            The folder will be created with the necessary access rights to the user, so subsequent installs do not require root access.
+
+            Do not run the toolchain-manager installation as root (for example, using `sudo`), as this would cause the directory to only grant access to root, meaning subsequent installations will also require root access.
```

Comment:
```suggestion
            Do not run the toolchain manager installation as root (for example, using ``sudo``), as this would cause the directory to only grant access to root, meaning subsequent installations will also require root access.
```